js/helpers/commonmark.js

```diff
@@ -1,9 +1,9 @@
 import markdownit from 'markdown-it';
 import { options } from 'markdown-it/lib/presets/commonmark';
 
+options.linkify = true;
+const markdown = markdownit(options);
 
 export default function(text) {
```

Comment:
It's not working like that. In ES6 file, there is no default. This is only for legacy files, this is why only handlebars helpers (meant to disappears) here need to use `.default()`.
Changing the export syntax won't change a thing :/